src/popup/components/TransactionList.vue

```diff
@@ -1,16 +1,15 @@
 <template>
-  <IonContent class="ion-padding ion-content-bg--lighter transaction-list">
```

Comment:
Because it was making the background of the transaction list lighter than the background of the actual page that holds the transaction list and other stuff. I compared the figma designs for this purpose. DId I break something by removing this class?

Reply:
It seems nothing broke, and it does indeed fix what you're describing. It was added to match the color of the `account-details page` but now it works there as well